extensions/INTEL/SPV_INTEL_long_composites.asciidoc

```diff
@@ -0,0 +1,249 @@
+SPV_INTEL_long_composites
+=========================
+
+Name Strings
+------------
+
+SPV_INTEL_long_composites
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Mariya Podchishchaeva, Intel
+- Alexey Sotkin, Intel
+- Ben Ashbaugh, Intel
+- Alexey Sachkov, Intel
+
+Notice
+------
+
+Copyright (c) 2023 Intel Corporation. All rights reserved.
+
+Status
+------
+
+* Shipping.
+
+Version
+-------
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2023-03-22
+| Revision           | 1
+|========================================
+
+Dependencies
+------------
+
+This extension is written against the SPIR-V Specification,
+Version 1.6 Revision 2, Unified
+
+This extension requires SPIR-V 1.0.
+
+Overview
+--------
+
+This extension adds new capability and instructions to allow to represent
+composites with number of Constituents greater than the maximum
+possible WordCount.
+
+Extension Name
+--------------
+
+To use this extension within a SPIR-V module, the following
+*OpExtension* must be present in the module:
+
+----
+OpExtension "SPV_INTEL_long_composites"
+----
+
+Modifications to the SPIR-V Specification, Version 1.5
+------------------------------------------------------
+Capabilities
+~~~~~~~~~~~~
+
+Modify Section 3.31, "Capability", adding these rows to the Capability table:
+
+--
+[options="header"]
+|====
+2+^| Capability ^| Implicitly Declares
+| 6089 | *CapabilityLongCompositesINTEL* +
+Allow to use *OpTypeStructContinuedINTEL*, *OpConstantCompositeContinuedINTEL*,
+*OpCompositeConstructContinuedINTEL* and *OpSpecConstantCompositeContinuedINTEL* instructions |
+|====
+--
+
+Instructions
+~~~~~~~~~~~~
+In section 3.42.6. Type-Declaration Instructions add the new instruction
+[cols="3", width="100%"]
+|=====
+2+^|*OpTypeStructContinuedINTEL* +
```

Comment:
```suggestion
2+|*OpTypeStructContinuedINTEL* +
```

The contents of these tables are centered right now.  I think this will fix it?  Same goes for the other instructions added by this extension, too.